cmd/action/mux.go

```diff
@@ -43,8 +43,9 @@ func NewMux() *Mux {
 }
 
 var (
-	defaultMux = NewMux()
-	inMemory   = regexp.MustCompile("^file:.*:memory:$|:memory:|^file:.*mode=memory.*")
+	// DefaultMux is the default Mux that is used by the different commands.
+	DefaultMux = NewMux()
```

Comment:
I don't like the fact that this is exported only to do the test. Can we do this different (by renaming the test package or rewrite the test)?

Reply:
e.g. calling `NewMux()` is the same as `defaultMux` (as of now).

Reply:
> I don't like the fact that this is exported only to do the test. Can we do this different (by renaming the test package or rewrite the test)?

Currently, the different commands depend on it and use it, but there are no tests for it. I'd structure it differently, but I can't add a piece of code without testing it. 

> e.g. calling NewMux() is the same as defaultMux (as of now).

Not really, because in the provider.go file, the `Register` is called on init.